module/localisation/BallLocalisation/src/BallModel.hpp

```diff
@@ -93,7 +93,7 @@ namespace module::localisation {
         inline int getRogueCount() const {
             return n_rogues;
         }
-        Eigen::Matrix<Scalar, 2, 1> getRogueRange() const {
+        [[nodiscard]] Eigen::Matrix<Scalar, 2, 1> get_rogue() const {
             return resetRange;
```

Comment:
Pretty sure this rogue range should also be just generating rogues with the correct distribution